src/InstagramClone.Api/Configuration/UserConfiguration.cs

```diff
@@ -0,0 +1,66 @@
+﻿using InstagramClone.Api.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api.Configuration
+{
+    public static class UserConfiguration
+    {
+
+        public static void SeedUser(this ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<User>()
+                .HasData(
+                    new User
+                    {
+                        Id = new  Guid("50ED31B9-30AE-4E7A-BEEF-5997EABA8A21"),// 1
+                        Email = "ehuerikenbaba@gmail.com",
+                        Password = "12345",
+                        FirstName = "Ehueriken",
+                        LastName = "Obuse",
+                        Avatar = "Sammy.jpeg",
+                        Posts = new List<Post> { new Post
+                        {
+                       
+                                Id = Guid.NewGuid(), 
+                                Content = "Hello world" ,
+                                AuthorId = new  Guid("50ED31B9-30AE-4E7A-BEEF-5997EABA8A21"),
+                                PostReactions = new  List<PostReaction>{ new PostReaction()
+                                    { Id = Guid.NewGuid(), 
+                                        CreatedDate = DateTime.Now,
+                                        AuthorId =  new Guid("50ED31B9-30AE-4E7A-BEEF-5997EABA8A21"),
+                                        Type = 3,
+                                        PostId =  new  Guid("50ED31B9-30AE-4E7A-BEEF-5997EABA8A21") 
```

Comment:
where have you defined the post with this Id? 

Every foreign key must link to a valid primary key in another table

Reply:
I am not sure if I understand your question thoroughly, but if I am to give a reply from what I understand I will say: I seeded the PostId in the user on line 25, as a new Guid